tests/Kernel/BlockTest.php

```diff
@@ -30,6 +30,7 @@ class BlockTest extends EntityKernelTestBase {
     'image',
     'breakpoint',
     'responsive_image',
+    'oe_theme_helper',
```

Comment:
Why?

Reply:
When installing the theme we install all the image styles. Most of the schema for this image styles comes from the image module (already there) but the new retine_image_scale is defined on the oe_theme_helper, hence the need to enable it as well.